iocore/net/quic/QUICTransportParameters.h

```diff
@@ -61,7 +61,7 @@ class QUICTransportParameterId
   }
 
   bool
-  operator==(uint16_t &x) const
+  operator==(const uint16_t &x) const
```

Comment:
Nitpick:  you could simply pass by value here and it would inline to the same asm code:  https://godbolt.org/z/9za8xz .

Reply:
I guess you can find a lot of this kind of stuff in QUIC module. It'd be great if you could make PRs to correct use of references. This change is fine with me because it's a right thing to do in terms of const-ify.